visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -29,6 +29,7 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 	private previousScaling: Vector3 = new Vector3(1, 1, 1)
 	private lineCount = 1
 	private nodeHeight = 0
+	private HEIGHT_OFFSET = 1
```

Comment:
I would expect an offset to be added to something, not multiplied. How about `HEIGHT_BASE_FACTOR` instead?

Reply:
I agree while it seems to only be read once and never changed. Therefore, we could just remove the variable again.

Reply:
Yes, I will push a changed version later today.
I tried something out and this is a remainder from it